backend/tests/mcp_server/test_interactive_form_question.py

```
@@ -192,6 +192,110 @@ async def capture(task_id, subtask_id, question_data):
+            captured.update(question_data)
+
+        with patch(
+            "app.mcp_server.tools.interactive_form_question._notify_frontend",
+            side_effect=capture,
+        ):
+            result = await interactive_form_question(
+                token_info=token,
+                questions=[
+                    {
+                        "id": "confirm_logic",
+                        "input_type": "single_choice(question(确认需求逻辑是否正确？))",
```

Ruff (0.15.15)

[warning] String contains ambiguous `？` (FULLWIDTH QUESTION MARK). Did you mean `?` (QUESTION MARK)?

(RUF001)